release-notes/README.md

```diff
@@ -4,6 +4,21 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## January 24, 2022
+
+### All Unity packages
+
+The ACP Unity packages now use XCFrameworks in order to support hardware with the new Apple M1 architecture while maintaining support for existing Intel architecture. See the respective GitHub repositories for the updated installation instructions.
+
+**IMPORTANT:** Upgrading to XCFrameworks distribution requires Xcode 12.0 or newer
+
+Below is a list of the new versions for each Unity package:
+
+* [Unity ACPCore v1.0.1](https://github.com/adobe/unity-acpcore/blob/master/bin/ACPCore-1.0.1-Unity.zip)
```

Comment:
This comment applies for all extensions:
Would it be possible to have release tags instead of using the master branch for these links? This will ensure the link remains valid even after future releases when the zip will be replaced with a more up-to-date lib.

Reply:
Created the release and tag manually on Github for now and added the links. Might need to automate it using Github actions if we plan to have frequent releases.